cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java

```diff
@@ -330,6 +329,9 @@ private LeaderElection createLeaderElection() {
       public void leader() {
         LOG.info("Became leader for master services");
 
+        twillRunner = baseInjector.getInstance(TwillRunnerService.class);
+        twillRunner.startAndWait();
```

Comment:
Creating `TwillRunner` with the child injector and verified the changes on the cluster.